edi_voxel/data/data_voxel_uom.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2019 Tecnativa - Ernesto tejeda
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo noupdate="1">
+    <record id="uom.product_uom_unit" model="uom.uom">
+        <field name="voxel_code">Unidades</field>
+    </record>
+    <record id="uom.product_uom_kgm" model="uom.uom">
+        <field name="voxel_code">Kgs</field>
+    </record>
+    <record id="uom.product_uom_hour" model="uom.uom">
+        <field name="voxel_code">Horas</field>
+    </record>
+    <record id="uom.product_uom_meter" model="uom.uom">
+        <field name="voxel_code">Metros</field>
```

Comment:
Hi @ernestotejeda thanks for this module. Do you know if this module is only for spanish community ?

Reply:
Voxel group is a company that operates both in Spain and Italy: https://www.voxelgroup.net. That's why we sent it here, but this seems to be locally coded for Spain according instructions. For Italian use, I suppose this should switch.

Reply:
Ok, just an idea but edi with external applications, and its specific parts could go to an extra repository like 'edi-solution' to avoid to have a bloated repo here, what do you think ? At least in future versions.

Reply:
Well, in fact this is a mix of 2 things: the export format (Voxel XML) + communications to the external gateway (FTP this time). Each EDI has own components, but we lack an infrastructure in Odoo for putting this in  a generic way, so each one has developed their own. Fortunately v14/v15 seems to introduce EDI in core, and maybe we can split this in 2: OCA/edi-format and OCA/edi-gateway or similar, but for now, I don't see productive to do that split, and this repo is not the more bloated one (see OCA/sale-workflow for example).

Reply:
for communications consider https://github.com/OCA/server-tools/tree/12.0/attachment_synchronize and https://github.com/OCA/storage/tree/12.0/storage_backend_sftp

Reply:
Yeah, we considered them, but in this case the layer to put over them would be higher than directly coding, due to strict filenames and how the errors detection is done (matching files in a folder against those sent into another).